test/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.test.ts

```diff
@@ -240,6 +240,49 @@ describe('VideoPriorityBasedPolicy', () => {
       received = policy.chooseSubscriptions();
       expect(received.array()).to.deep.equal([1, 3, 5, 7, 9, 11, 13, 15, 17, 19]);
     });
+
+    it('pause tiles if not enough bandwidth for default preference', () => {
+      const policy = new VideoPriorityBasedPolicy(logger);
+      policy.bindToTileController(tileController);
+      updateIndexFrame(videoStreamIndex, 3, 0, 1000);
+      policy.updateIndex(videoStreamIndex);
+      let resub = policy.wantsResubscribe();
+      expect(resub).to.equal(true);
+      let received = policy.chooseSubscriptions();
+      expect(received.array()).to.deep.equal([2, 4]);
+      const tile1 = tileController.addVideoTile();
+      tile1.stateRef().boundAttendeeId = 'attendee-1';
+      const tile2 = tileController.addVideoTile();
+      tile2.stateRef().boundAttendeeId = 'attendee-2';
+      expect(tileController.getAllVideoTiles().length).to.equal(3);
+      const tile3 = tileController.getAllVideoTiles()[0];
+      expect(tile3.state().boundAttendeeId).to.equal('attendee-3');
+
+      const metricReport = new DefaultClientMetricReport(logger);
+      metricReport.globalMetricReport = new GlobalMetricReport();
+      metricReport.globalMetricReport.currentMetrics['googAvailableReceiveBandwidth'] = 3000 * 1000;
+      policy.updateMetrics(metricReport);
+
+      resub = policy.wantsResubscribe();
+      expect(resub).to.equal(true);
+      received = policy.chooseSubscriptions();
+      expect(received.array()).to.deep.equal([2, 4, 6]);
```

Comment:
For learning: What do these [2,4,6] denote?

Reply:
It is the streamId.  We have `updateIndexFrame` function to create a bunch of streamIds for each attendee.
This test basically means subscribing to video stream 2,4,6 (2 for attendee-1, 4 for attendee-2, and 6 for attendee-3).